lifecycle-operator/controllers/common/helperfunctions.go

```diff
@@ -105,3 +106,11 @@ func getObject(k8sclient client.Client, log logr.Logger, ctx context.Context, de
 	}
 	return nil
 }
+
+func GetControllerInfo(req ctrl.Request) map[string]interface{} {
```

Comment:
let's make it map[string] string instead

Reply:
Alright I will that.